content/docs/examples/multicluster/icp/index.md

```diff
@@ -54,6 +54,10 @@ in the cluster. You need two steps:
 
 1.  Add IP routers from `cluster-2` to `cluster-1`.
 
+{{< warning >}}
+This works if all the nodes among the multiple IBM Cloud Private clusters are located in the same subnet, However, it isn't able to add BGP routers directly for the nodes that are located in different subnets, because the IP addresses cannot be reached in just a hop. In that case, the solution is setting up a VPN for pod communication across clusters, please refer to [this article](https://medium.com/ibm-cloud/setup-pop-to-pod-communication-across-ibm-cloud-private-clusters-add0b079ebf3) for more details.
```

Comment:
```suggestion
This approach works if all the nodes within the multiple IBM Cloud Private clusters are located in the same subnet. It is unable to add BGP routers directly for nodes located in different subnets because the IP addresses must be reachable with a single hop. Alternatively, you can use a VPN for pod communication across clusters. Refer to [this article](https://medium.com/ibm-cloud/setup-pop-to-pod-communication-across-ibm-cloud-private-clusters-add0b079ebf3) for more details.
```